core/src/impl/Kokkos_Combined_Reducer.hpp

```diff
@@ -247,6 +249,22 @@ struct CombinedReducerImpl<std::integer_sequence<size_t, Idxs...>, Space,
 
      ...);
   }
+
+  template <int Idx, class View>
+  KOKKOS_FUNCTION void write_one_value_back_on_device(
```

Comment:
Converted to a `static` function in [a28463e](https://github.com/kokkos/kokkos/pull/5727/commits/a28463eaae89518a32405a9df9e9cb0394bb3d55).
I recall needing to remove `static` from that function at one point, because something was unable to work with a device static function at that time. But that may no longer be the case now. The CI will confirm this.